ios/Classes/TiBarcodeModule.mm

```diff
@@ -309,7 +309,10 @@ -(void)capture:(id)args
 			[controller setSoundToPlay:soundURL];
 		}
 	}
-	
+    if(([UIDevice currentDevice].orientation == UIDeviceOrientationLandscapeLeft) || ([UIDevice currentDevice].orientation == UIDeviceOrientationLandscapeRight)||([UIDevice currentDevice].orientation == UIDeviceOrientationFaceUp) || ([UIDevice currentDevice].orientation == UIDeviceOrientationFaceDown)|| ([UIDevice currentDevice].orientation == UIDeviceOrientationUnknown)|| ([UIDevice currentDevice].orientation == UIDeviceOrientationPortraitUpsideDown))
+    {
+        [[UIDevice currentDevice] setValue:[NSNumber numberWithInteger: UIInterfaceOrientationPortrait] forKey:@"orientation"];
+    }
```

Comment:
Not sure if that is a proper fix.

- You are checking all possible orientations instead of passing the orientation directly
- You changes the external ZXing library to force the device to not rotate (what if people want it rotating?)

In general, the UI should rather change to adopt the new orientation (e.g. a view with `bottom: 10` should still be visible).